README.md

```diff
@@ -62,22 +60,16 @@ If a loader is already registered, this will simply return `true`.
 
 MIT
 
-[interpret]: http://github.com/gulpjs/interpret
-[Liftoff]: http://github.com/gulpjs/liftoff
-
-[downloads-image]: http://img.shields.io/npm/dm/rechoir.svg
+<!-- prettier-ignore-start -->
+[interpret]: https://github.com/gulpjs/interpret
+[require.extensions]: https://nodejs.org/api/modules.html#modules_require_extensions
+[liftoff]: https://github.com/js-cli/js-liftoff
```

Comment:
I've been trying to move these to the bottom of the URL block so the upgrade tool has an easier time diffing. They could even exist in their own `<!-- prettier-ignore-start -->` block at the bottom.

Reply:
Got it.

Reply:
Got it.